### PR TITLE
Create CONTRIBUTING

### PR DESCRIPTION
**Pull request by Smitholi67**
Centralizing information about how to contribute to CNTi.  Once this is merged, the README files in each of our repos will be updated to point to this document.  This will make it easier to maintain and ensure more consistency.

### CODE REVIEW

File: CONTRIBUTING [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+Welcome to the Cloud Native Telecom Initiative (CNTi). This is an open, public project welcoming anyone who would like to help in one or more of our focus areas.  Our goal is to advance cloud native networking including developing best practices, maintaining a testing catalog, and providing vendor neutral certification.  We're glad you're here!
+
+To learn more about our focus areas and our different collaboration tools, please visit our focus area wiki pages:
+
+- [Best Practices](https://wiki.lfnetworking.org/x/HAAxBw)
+
+- [Test Catalog](https://wiki.lfnetworking.org/x/HgAxBw)
+
+- [Certification](https://wiki.lfnetworking.org/x/IAAxBw)
+
```
Comment (martin-mat) [inline CONTRIBUTING:12]: Do we want to include crystal libs?
https://github.com/cnf-testsuite/
Often we have direct test functionality dependencies there.

File: CONTRIBUTING [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+### Join our Meetings
+
+Absolutely everyone is welcome to come to any of our meetings. You never need an invite to join us. In fact, we want you to join us, even if you don’t have anything you feel like you can  contribute. Just being there is a good start!
+
+Over time, we hope that you feel comfortable voicing your opinions, ideas and providing feedback with the community. Sharing your own ideas, and experiences can help in ways you might not initially realize.
+
+Check the respective focus area wikis for more details on when we meet for [Best Practices](https://wiki.lfnetworking.org/x/HAAxBw), [Test Catalog](https://wiki.lfnetworking.org/x/HgAxBw), and [Certification](https://wiki.lfnetworking.org/x/IAAxBw). 
+
+### Join a GitHub Discussion
```
Comment (martin-mat) [inline CONTRIBUTING:54]: additional section - "Join CNTi slack channels" ...?

File: CONTRIBUTING [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+Reviews and comments on open Pull Requests are always appreciated.  A minimum of 3 community approvals, from at least 2 different companies, are required to merge PRs in any of our repositories -  so your review is greatly appreciated.
+
+Where a comment is better expressed as a proposed change, the change can be made directly either by editing the file (right hand "..." -> "edit file" for larger changes, or the "document-with-+-" icon above the comment window for smaller ones).   The author can accept these changes directly, which is much easier for them than writing a change themselves.
+
+#### Steps for Creating a New PR
+
+- Once you have made your change or added new content
+- Ensure you are up to date with the `main` branch of the cnf-wg repository
+- Open a new Pull Request in the repository of the correct Focus Area: i.e. [Best Practices](https://github.com/lfn-cnti/bestpractices/pulls), [Test Catalog](https://github.com/cnti-testcatalog/testsuite/pulls), or [Certification](https://github.com/lfn-cnti/certification/pulls)
```
Comment (martin-mat) [inline CONTRIBUTING:99]: I suggest to list all repos just on one place in this document.

File: CONTRIBUTING [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+#### Steps to Accept a PR
+
+1. Reviewers will review the PR and give their feedback: approve, request changes, or comment w/o approval
+1. After the required number of approvals from reviewers is reached the PR may be merged
+
+Anyone with merge access can merge the PR after the item has been approved.
+
+### Acceptance Criteria for Contributions
+
+Reviewers - Any community member
```
Comment (martin-mat) [inline CONTRIBUTING:111]: Any contributor of respective repository

**Review by martin-mat — CHANGES_REQUESTED** [review]

File: CONTRIBUTING (hunk 1 above)
Comment (Smitholi67) [reply CONTRIBUTING:12]: Good point, however, the links are pointing to the wiki page for respective focus areas.  On those wiki pages we have a link to our repositories.  I would suggest we update this page to include the other repos where we have dependencies for test suite.  See here -> https://wiki.lfnetworking.org/display/LN/Repositories+and+Project+Backlogs

File: CONTRIBUTING (hunk 2 above)
Comment (Smitholi67) [reply CONTRIBUTING:54]: Another good idea, but I feel it should be sufficient with the links to the focus area wiki pages as they include the relevant Slack channels for each.

File: CONTRIBUTING (hunk 4 above)
Comment (Smitholi67) [reply CONTRIBUTING]: Agreed.  New commit made.

**Review by rich-l — APPROVED** [review]
lgtm

**Review by taylor — APPROVED** [review]
lgtm as first version

**Comment by cassamajor** [comment]
Please update the filename to `CONTRIBUTING.md`, otherwise the document will lack Markdown formatting.

**Review by cassamajor — CHANGES_REQUESTED** [review]
Please update the filename to `CONTRIBUTING.md`, otherwise the document will lack Markdown formatting.

**Review by martin-mat — APPROVED** [review]

**Review by taylor — APPROVED** [review]

**Pull request merged**